blocks/header/header.css

```diff
@@ -1,809 +1,268 @@
   font-family: var(--ff-volvo-novum);
-  border: 0;
+  font-size: 14px;
+  line-height: 125%;
+  letter-spacing: 0.4px;
+  border-bottom: solid 2px transparent;
+  padding: 12px 0;
 }
 
-header nav .sections .section > a {
-  padding: 15px;
-  border-bottom: 1px solid var(--c-grey-1);
-  text-transform: uppercase;
+.header--hamburger-open .header__link:hover,
+.header--hamburger-open .header__link:focus,
+.header--hamburger-open .header__link:active {
+  border-bottom: solid 2px transparent;
 }
 
-header nav .sections .section a::after {
-  position: absolute;
-  right: 15px;
-  font-family: var(--ff-fontawesome);
-  content: '\f105';
-  font-size: 20px;
-  width: 8px;
-  height: 8px;
-  margin-top: -4px;
+.header__link:any-link:active {
+  color: var(--c-main-black);
 }
 
-header nav .sections .section li > *:not(a.primary-link, ul) {
-  /* hide all elements in list items but the primary link and any sub list */
-  display: none;
+.header__link .icon,
+.header__link .icon svg {
+  height: 24px;
+  width: auto;
 }
 
-/* 1st level */
-header nav .sections .section > ul > li a {
-  padding: 15px 0 30px;
+.header__actions {
+  display: flex;
+  gap: 24px;
 }
 
-header nav .sections .section > ul > li.overview > *,
-header nav .sections .section > ul > li > p.title {
-  display: block;
-  margin: 0;
+.header__actions-list,
+.header__main-links {
+  display: flex;
+  transition: var(--menu-animation);
+  right: max(-375px, -95vw);
+  position: absolute;
+  width: 95vw;
+  max-width: 375px;
+  background: var(--c-white);
 }
 
-header nav .sections .section > ul > li a::after {
-  content: '\f067';
-  font-size: 14px;
-  margin-top: 0;
+.header__actions-list {
+  list-style: none;
+  margin: 0;
+  gap: 24px;
+  border-bottom: solid 1px var(--c-grey-300);
+  top: 0;
+  padding: 16px;
 }
 
-header nav .sections .section.expand > a::after {
-  content: '\f104';
-  right: unset;
-  left: 0;
+.header__action-link.header__hamburger-menu {
+  padding: 0;
+  margin: 0;
+  background: inherit;
+  border: none;
+  justify-content: center;
 }
 
-header nav .sections .section > ul > li.overview a::after {
-  content: unset;
+.header__main-links {
+  top: 102px;
+  padding: 0 16px;
+  height: calc(100vh - 102px);
 }
 
-/* 2nd and deeper levels */
-header nav .sections .section > ul > li.expand > ul {
-  padding-left: 15px;
+.header__action-link {
+  display: flex;
+  flex-direction: column;
 }
 
-header nav .sections .section > ul > li > ul > li a {
-  padding-top: 0;
+.header__mobile-actions {
+  display: flex;
+  gap: 16px;
 }
 
-header nav .sections .section > ul > li.expand a::after {
-  content: '\f068';
+.header--hamburger-open .header__menu-background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: rgb(20 20 20 / 40%);
+  width: 100vw;
+  height: 100vh;
+  display: block;
 }
 
-header nav .sections .section > ul > li.expand > ul li a::after {
-  content: unset;
+.header--hamburger-open .header__actions-list {
+  right: 0;
+  top: 0;
```

Comment:
You can remove `top:0 ` have same css rule at 132

Reply:
Removed